contribute/contribution-guide.md

```diff
@@ -59,6 +59,47 @@ To contribute code to Apache Beam, you’ll have to do a few administrative step
 
 ### One-time Setup
 
+#### Intellij
+
```

Comment:
How about filing a starter JIRA issue to fill this in and linking to that? Helps encourage folks who might know to come back and contribute.